ddcz/templates/base/historic/left_menu.html

```diff
@@ -0,0 +1,31 @@
+<nav id="left_menu">
+    <ul class="nav_links">
+        <li><a class="text-gold" href="">Chat</a></li>
```

Comment:
Other classes are with underscore `_`, any reason to go with dash suddenly `-`?

Reply:
Also, I think classes should describe semantic values maybe...? `menu-item-gold` rather than `text-gold`?

Reply:
It is not only for the menu though.